iterators.go

```diff
@@ -0,0 +1,84 @@
+package mijn_host
+
+import (
+	"iter"
+
+	"github.com/libdns/libdns"
+	"github.com/pbergman/libdns-mijn-host/client"
+)
+
+type ResourceRecordsIterator interface {
+	Iterate() iter.Seq[libdns.RR]
+}
+
+func MarshallRecord(zone string, record *libdns.RR) *client.DNSRecord {
+	var x = &client.DNSRecord{
+		Type:  record.Type,
+		Value: record.Data,
+		Name:  libdns.AbsoluteName(record.Name, zone) + ".",
```

Comment:
When libdns/libdns@fce8af8a49e712e0b5a59238f0cfa3e3c4c88bed is released, `libdns.AbsoluteName` will always return a domain with a trailing dot (and will right now if the zone has a trailing dot), so this line will make `Name` end with 2 dots.